Source/x11/XGServerWindow.m

```diff
@@ -4409,75 +4415,140 @@ - (void) freecursor: (void*) cid
   return depth;
 }
 
+/* This method assumes that we deal with one X11 screen - `defScreen`.
+   Basically it means that we have DISPLAY variable set to `:0.0`.
```

Comment:
No, even if there could only be one screen, both numbers could be set to arbitrary values.

Reply:
You're right it's not clear description. I mean for single application copy we have signle display and scree. I'll change desciption.

Reply:
Done.